src/module.c

```diff
@@ -1447,6 +1451,46 @@ int populateArgsStructure(struct redisCommandArg *args) {
     return count;
 }
 
+/* RedisModule_AddACLCategory can be used to add new ACL command categories. Category names
+ * can only contain alphanumeric characters, underscores, or dashes. Categories can only be added
+ * during the RedisModule_OnLoad function. Once a category has been added, it can not be removed. 
+ * Any module can register a command to any added categories using RedisModule_SetCommandACLCategories.
+ * 
+ * Returns:
+ * - REDISMODULE_OK on successfully adding the new ACL category. 
+ * - REDISMODULE_ERR on failure.
+ * 
+ * On error the errno is set to:
+ * - EINVAL if the name contains invalid characters.
+ * - EBUSY if the category name is already in use.
```

Comment:
```suggestion
 * - EBUSY if the category name was already created.
```

Reply:
I originally thought wording like this was confusing since it will also throw the error for one of Redis' built in categories.

Reply:
I think `already in use` explains it better in case of built in categories and the ones added by the modules.

Reply:
"in use" sounds to me like "you can't use it", when in fact i actually expect modules to ignore this error and proceed.

Reply:
`if the category name already exists.` ?

Reply:
yeah, that's even better.. describes exactly what we do..